dashboard/http_server_agent.py

```diff
@@ -10,7 +11,7 @@
 
 from ray.dashboard.optional_deps import aiohttp, aiohttp_cors, hdrs
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(ray_constants.AGENT_LOGGER_NAME)
```

Comment:
Confirming these agents are all initialized from root `dashboard/agent.py`? If so , doesn't 
```
logger = logging.getLogger()
```
have the same effect? (I think it inherits the root/parent logger)

Reply:
Yeah the reason for this is totally unclear as written, I'll add a comment in the code.  It's because if I use the root logger, then all the runtime env installation logs will automatically be also sent to the root logger, which means they'll show up in `dashboard_agent.py`--which we explicitly don't want.  We only want them to show up in the `runtime_env_setup-<job-id>.log` files.

I might be missing another way to do this though, let me know if you know of one.